src/settings/JobProfiles/JobProfilesForm/JobProfilesForm.js

```diff
@@ -0,0 +1,132 @@
+    mappingProfiles,
+  } = props;
+
+  return (
+    <FormattedMessage id="ui-data-export.mappingProfiles.newProfile">
+      {contentLabel => (
+        <Layer
+          isOpen
+          contentLabel={contentLabel}
+        >
+          <FullScreenForm
+            id="job-profiles-form"
+            paneTitle={<FormattedMessage id="ui-data-export.jobProfiles.newProfile" />}
+            isSubmitButtonDisabled={pristine || submitting}
+            onSubmit={handleSubmit}
+            onCancel={onCancel}
+          >
+            {!hasLoaded && <Preloader />}
+            {hasLoaded && (
+              <div className={css.jobProfilesFormContent}>
+                <div data-test-job-profile-form-name>
+                  <Field
+                    label={<FormattedMessage id="stripes-data-transfer-components.name" />}
+                    name="name"
+                    id="job-profile-name"
+                    component={TextField}
+                    fullWidth
+                    required
+                  />
+                </div>
+                <div data-test-job-profile-form-mapping-profile>
+                  <Field
+                    label={<FormattedMessage id="ui-data-export.mappingProfile" />}
+                    name="mappingProfileId"
+                    id="mapping-profile-id"
+                    component={Select}
+                    dataOptions={mappingProfiles}
+                    placeholder={intl.formatMessage({ id: 'ui-data-export.selectMappingProfile' })}
+                    fullWidth
+                    required
+                  />
+                </div>
+                <div data-test-job-profile-form-protocol>
+                  <Field
+                    label={<FormattedMessage id="ui-data-export.protocol" />}
+                    name="protocol"
+                    id="job-profile-protocol"
+                    component={Select}
+                    dataOptions={[]}
+                    fullWidth
+                  />
+                </div>
+                <div data-test-job-profile-description>
+                  <Field
+                    label={<FormattedMessage id="ui-data-export.description" />}
+                    name="description"
+                    id="job-profile-description"
+                    component={TextArea}
+                    fullWidth
+                  />
+                </div>
+              </div>
+            )}
+          </FullScreenForm>
+        </Layer>
+      )}
+    </FormattedMessage>
+  );
+};
+
+JobProfilesForm.propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  onCancel: PropTypes.func,
+  intl: PropTypes.object.isRequired,
+  pristine: PropTypes.bool.isRequired,
+  submitting: PropTypes.bool.isRequired,
+  mappingProfiles: PropTypes.arrayOf(PropTypes.object),
+  hasLoaded: PropTypes.bool,
+};
+
+JobProfilesForm.defaultProps = {
+  onCancel: noop,
+  hasLoaded: false,
+  mappingProfiles: [],
+};
+
+export default injectIntl(stripesFinalForm({
```

Comment:
Why you don't want to use `useIntl()` hook instead of `injectInlt()` here? I think in this case `intl` prop could be omitted

Reply:
Done

Reply:
@Vladyslav-Velytskyi Unfortunately these changes cause some unexpected errors that require some time to investigate. So it was agreed to use an old syntax without hooks and plan to investigate this issue in Q3. 
Anyway, this is a very good syntax improvement and it will be cool to apply it in the future. Thanks.